java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -203,4 +203,25 @@ public interface StringCommands {
      * }</pre>
      */
     CompletableFuture<Long> strlen(String key);
+
+    /**
+     * Overwrites part of the string stored at <code>key</code>, starting at the specified <code>
+     * offset</code>, for the entire length of <code>value</code>.<br>
+     * If the <code>offset</code> is larger than the current length of the string at <code>key</code>,
+     * the string is padded with zero-bytes to make <code>offset</code> fit. Creates the key if it
+     * doesn't exist.
+     *
+     * @see <a href="https://redis.io/commands/setrange/">redis.io</a> for details.
+     * @param key The key of the string to update.
+     * @param offset The position in the string where <code>value</code> should be written.
+     * @param value The string which should be written at the position specified by <code>offset
+     *     </code>.
+     * @return The length of the string stored at <code>key</code> after it was modified.
+     * @example
+     *     <pre>{@code
+     * long len = client.setrange("key", 6, "Redis").get();
```

Comment:
We could make this example a little better. 
1. Can you add a comment that says it will create a new string if `key` doesn't exist. 
2. Can you call client.get(key) and print that out to say the result will be "\x00\x00\x00\x00\x00\x00Redis"

Reply:
added